Makefile

```diff
@@ -136,6 +136,15 @@ docker-build: test ## Build docker image with the manager.
 docker-push: ## Push docker image with the manager.
```

Comment:
Unrelated to this PR we should consider finding a way to build and push images outside of a Makefile